lambdas/death-enrichment/src/test/java/uk/gov/di/data/lep/DeathEnrichmentTest.java

```diff
@@ -49,6 +51,7 @@ static void setup(){
     @BeforeEach
     void refreshSetup() {
         reset(objectMapper);
+        clearInvocations(awsService);
```

Comment:
we seem to use both reset and clearInvocations in the codebase - probably want a consistent approach, and I think that's clearInvocations for awsService, but wanted to check before updating them all

Reply:
I'm not sure which is better here, but I think lets split fixing this/making it consistent somewhere else/in a different ticket.

They do achieve different things, so it's not completely unreasonable for us to have both.